internal/tools/tools.go

```diff
@@ -15,10 +15,12 @@ import (
 	_ "github.com/frapposelli/wwhrd"
 	_ "github.com/fzipp/gocyclo"
 	_ "github.com/go-enry/go-license-detector/v4/cmd/license-detector"
+	_ "github.com/golang/mock/mockgen"
```

Comment:
This is added twice, since you install it from the `proto` Go module I think we can remove it from here
```suggestion
```

(and run `go mod tidy` afterwards)